Library/Formula/euslisp.rb

```diff
@@ -0,0 +1,39 @@
+class Euslisp < Formula
+  desc "EusLisp is an integrated programming system for the research on intelligent robots"
+  homepage "https://github.com/euslisp/EusLisp"
+  url "https://github.com/euslisp/EusLisp/archive/EusLisp-9.15.tar.gz"
+  version "9.15"
+  sha256 "74cf37606bd9fcb38ddcd6dcfbbb0989375faa9c18ada3dff6e39d610d966ea9"
+  head "https://github.com/euslisp/EusLisp.git", :using => :git
+
+  def install
+    ENV.deparallelize
+
+    prefix.install Dir["contrib"], Dir["doc"], Dir["lib"], Dir["lisp"], Dir["models"]
+
+    cd "#{prefix}"
+
+    ENV["ARCHDIR"] = "Darwin"
+    ENV["EUSDIR"] = Dir.pwd
+    ENV["PATH"] = "#{ENV["PATH"]}:#{ENV["EUSDIR"]}/Darwin/bin"
```

Comment:
You can just do `ENV.prepend_path "PATH", directory/"bin"` here. Directory ideally being `buildpath` but otherwise `prefix`.